### PR TITLE
feat: implement reconciler retrieve ingestion logs

### PR DESCRIPTION
**Pull request by leoparente**

### CODE REVIEW

**Comment by leoparente** [comment]
Still missing parse entity

**Review by mfiedorowicz — APPROVED** [review]

File: diode-server/reconciler/logs_retriever.go [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+)
+
+type extraAttributesWrapper struct {
+	ExtraAttributes string `json:"$"`
+	IngestionTs     string `json:"ingestion_ts"`
+}
+type redisLogResult struct {
+	ExtraAttributes extraAttributesWrapper `json:"extra_attributes"`
+	ID              string                 `json:"id"`
+	Values          []interface{}          `json:"values"`
```
Comment (mfiedorowicz) [inline diode-server/reconciler/logs_retriever.go]: Do we need this at all?

Comment (leoparente) [reply diode-server/reconciler/logs_retriever.go]: I don't think so, I will remove it as well. I've already removed some unused fields

Comment (mfiedorowicz) [reply diode-server/reconciler/logs_retriever.go]: Probably we can remove from test inputs as well

Comment (leoparente) [reply diode-server/reconciler/logs_retriever.go]: I kept on log to mock the full data from the redis FT SEARCH request

**Pull request merged**